### PR TITLE
P1 rating

### PR DESCRIPTION
**Pull request by anshulraheja**
Rating Component

### CODE REVIEW

File: components/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+:root {
+    --RATING-CHECKED-COLOR: #FFA500;
+    --RATING-UNCHECKED-COLOR: #DDD;
+}
+.rating {
+    display: inline-flex;    
+}
+
+.rating_input {
+    position: absolute !important;
```
Comment (muskanbagrecha) [inline components/rating/rating.css]: Try not to use important unless absolutely required.

**Comment by ritikaawasthy** [comment]
looks good to me

File: components/rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    left: -9999px !important;
```
Comment (muskanbagrecha) [inline components/rating/rating.css]: I see you are trying to hide the radio input. Can you perhaps try display property?

**Review by muskanbagrecha — CHANGES_REQUESTED** [review]

File: components/rating/rating.css (hunk 2 above)
Comment (anshulraheja) [reply components/rating/rating.css]: Yes, my bad. Thank you!

**Pull request merged**